syntheticsclientv2/common_models.go

```diff
@@ -303,6 +304,7 @@ type HttpCheckV2Input struct {
 		UserAgent          *string         `json:"userAgent"`
 		Verifycertificates bool            `json:"verifyCertificates"`
 		HttpHeaders        []HttpHeaders   `json:"headers,omitempty"`
+		Validations        []Validations   `json:"validations,omitempty"`
```

Comment:
Same thing here.

Also would you mind updating the integration test json bodies for http checks?
`createHttpCheckV2Body`
and
`updateHttpCheckV2Body`

are used with the `TestLiveHttpCheckCreateUpdateDeleteV2` integration test

I run those with `go test -tags=integration -test.run=TestLiveHttpCheckCreateUpdateDeleteV2`